suitcase/spec.py

```diff
@@ -325,10 +372,7 @@ def __len__(self):
     def __eq__(self, obj):
         if not isinstance(obj, type(self)):
             return False
-        return (self.specfile == obj.specfile and
-                self.raw_scan_data == obj.raw_scan_data and
-                self.md == obj.md and
-                self.scan_data.equals(obj.scan_data))
+        return _md5(self.raw_scan_data) == _md5(obj.raw_scan_data)
```

Comment:
I'm not really sure how this is going to be used in the wild.  While you are absolutely correct that these calls are probably expensive, I'm hesitant to do any sort of caching of this _md5 call for performance reasons until someone complains about the performance being slow.  That being said, that these Specscan objects might be slow could maybe encourage people to *not* use them and do everything in the ophyd/bluesky/databroker stack instead